.github/util/package-install4j.sh

```diff
@@ -35,13 +35,14 @@ else
   # sudo apt install -y $INSTALL4J_CACHE/install4j_linux-x64_10_0_7.deb
 
   # installer automation for macos-latest; macos needed for apple notarizing
-  wget -nv --directory-prefix=$INSTALL4J_CACHE -nc https://download.ej-technologies.com/install4j/install4j_macos_10_0_7.dmg
-  sleep 5
-  hdiutil attach /Users/runner/.install4j10/install4j_macos_10_0_7.dmg
-  sleep 5
-  cp -rf /Volumes/install4j/install4j.app /Applications
-  sleep 5
-  hdiutil detach /Volumes/install4j
+  # Commenting out code that was used to download and install Install4J on macOS:
+#  wget -nv --directory-prefix=$INSTALL4J_CACHE -nc https://download.ej-technologies.com/install4j/install4j_macos_10_0_7.dmg
+#  sleep 5
+#  hdiutil attach /Users/runner/.install4j10/install4j_macos_10_0_7.dmg
+#  sleep 5
+#  cp -rf /Volumes/install4j/install4j.app /Applications
+#  sleep 5
+#  hdiutil detach /Volumes/install4j
```

Comment:
Hm.. i believe this will break the build, as this in the install4j tool that is used to generate the mac _and_ windows installers.

Reply:
There’s no specific code in this section related to building a Windows installer, but at the bottom it is and hence commented out.